### DISCUSSION TITLE
Table is itself placing on top of page.

### DISCUSSION THREAD

**Discussion by jaads** [dolanmiu/docx#924] [General]
After upgrade to Version 6, my Tables are suddenly placing themselves to the top of the page. 

Image a new document, where its first child is a paragraph, representing a header, and the second child is a Table. On the final docx file, the table comes then before the paragraph.

It this a bug or is there a way to place the table at this place where it was declared, like it was in version 5.5.0?

**Comment by dolanmiu** [comment]
in v6, headers aren't being created by default, so naturally there will be more space now, to fix this, add a header explicitly, or increase the top-margin of the document

This is mentioned in:

https://github.com/dolanmiu/docx/pull/829

I think this is the reason for your issue anyway, if not, let me know.

**Reply by jaads** [reply]
Sorry I didn't mean a header, i meant a headline. But my document has in fact also a header. 
So when I insert the table, it places itself above the headline (meaning a paragraph with the `heading` option), although i declared it below the headline.

**Reply by dolanmiu** [reply]
Could you show me a code snippet for me to re-create?
